spec/models/sponsor_spec.rb

```diff
@@ -328,6 +328,24 @@
       expect(request_fulfilled_sponsor.eligible_for_sponsorship?).to eq false
     end
 
+    describe '#sponsorship_changed!', :mytest => true do
```

Comment:
By default, if any examples are tagged with `:focus`, only they will run when you run `$ rspec`. Perhaps it's a good practice to use this tag instead of any custom ones: Before submitting a PR, the entire test suite should be run locally. If only a small subset of tests get run when you do that, you know you've left the `:focus` tag somewhere.